GTG/gtk/backends/backendstree.py

```diff
@@ -190,8 +190,7 @@ def cell_edited_callback(self, text_cell, path, new_text):
         @param new_text: the new name of the backend
         """
         # we strip everything not permitted in backend names
-        new_text = ''.join(c for c in new_text if (c.isalnum() or
-                                                   c in [" ", "-", "_"]))
+        new_text = ''.join(c for c in new_text if (c.isalnum() or c in [" ", "-", "_"]))
```

Comment:
Same `if (...)` parenthesis thing like I've written before, but I am not sure if it would cause some ambiguity at syntax level

Reply:
It's small, I don't think it would cause confusion to remove them, but let me know if you want to leave them